common/process_test.cc

```diff
@@ -313,11 +313,10 @@ TEST_F(ProcessTest, LogOutputLevelDetection) {
 }
 
 TEST_F(ProcessTest, Terminate) {
+  // Use ping to simulate a sleep instead of timeout since timeout fails with
```

Comment:
That's a neat trick!

Reply:
It saddens me that this seems to be the only way on Windows. People also suggest the choice command, but that one also accepts user input, so I was afraid that it has the same issue.